cgi-bin/LJ/Poll/Question.pm

```diff
@@ -15,6 +15,26 @@ package LJ::Poll::Question;
 use strict;
 use Carp qw (croak);
 
+
+### I don't quite see why I should need to do the use here, but it does
+### seem to be necessary - asb
+use LJ::Poll::Question::CheckBox;
+use LJ::Poll::Question::RadioButton;
+use LJ::Poll::Question::MultiChoice;
+use LJ::Poll::Question::Scale;
+use LJ::Poll::Question::DropDown;
+use LJ::Poll::Question::Text;
+
```

Comment:
This is common and annoying enough that we have a shortcut:

`LJ::ModuleLoader->require_subclasses( "LJ::Poll::Question" );`